book/07-git-tools/sections/advanced-merging.asc

```diff
@@ -115,7 +115,7 @@ $ git status -sb
 
 The `git merge --abort` option tries to revert back to your state before you ran the merge. The only cases where it may not be able to do this perfectly would be if you had unstashed, uncommitted changes in your working directory when you ran it, otherwise it should work fine.
 
-If for some reason you find yourself in a horrible state and just want to start over, you can also run `git reset --hard HEAD` or wherever you want to get back to. Remember again that this will blow away your working directory, so make sure you don't want any changes there.
```

Comment:
Yes, this is synthetic and clear. Maybe we can also say that the known state reached is the last commit of the current branch.

Whats about this updated proposal?
> If for some reason you just want to start over, you can also run git reset --hard HEAD, and your repository will be back **to last commit state. Remember again that any uncommited work will be lost,** so make sure you don't want any changes there.